src/figuresClasses.ts

```diff
@@ -1,11 +1,97 @@
-export interface Figure {}
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
```

Comment:
Enum constant names should start with a capital letter. Consider changing 'red', 'green', and 'blue' to 'Red', 'Green', and 'Blue'.